apps/wallets/smart-wallet/next/src/app/page.tsx

```diff
@@ -14,6 +15,37 @@ export default function Home() {
     const { status: walletStatus } = useWallet();
     const [nftSuccessfullyMinted, setNftSuccessfullyMinted] = useState(false);
 
+    const renderActionButton = () => {
```

Comment:
<img width="336" alt="Screenshot 2024-09-08 at 9 02 51 PM" src="https://github.com/user-attachments/assets/40da0a6c-f9be-4571-a626-2cde9ad64fbd">

I'm not sure this is working as intended